test/jdk/javax/swing/Popup/TaskbarPositionTest.java

```diff
@@ -61,6 +61,7 @@
  * @key headful
  * @summary Tests the location of the heavy weight popup portion of JComboBox,
  * JMenu and JPopupMenu.
+ * On MAC disable all system shortcuts before executing this test case.
```

Comment:
Well, the test uses only one particular shortcut which was introduced in macOS Ventura.
I'd rather have a more specific statement, something like this: The test uses Ctrl+Down Arrow (↓) which is a system shortcut on macOS, disable it in system settings, otherwise the test will fail.

The systems which are exclusively used for running tests should have all system shortcuts disabled. It may be *unfeasible* to disable all the shortcuts on a system that you use regularly for doing your work as well as for running tests (while you're updating it or fixing a bug).

Reply:
Thanks @aivanov-jdk, that make more sense and I have updated.